apps/api/src/server/applications/application/documents/document.js

```diff
@@ -98,8 +98,17 @@ export const buildNsipDocumentPayload = (version) => {
  */
 const buildBlobUri = (containerName, path) => {
```

Comment:
Not necessarily one for this PR but I wonder if these sorts of functions would be better placed somewhere higher up in the codebase. My work in an Azure function could make use of this function.

Reply:
Absolutely, when you get to it - if you feel it's worth re-using - refactor it somewhere you can re-use it.